package/MDAnalysis/core/accessors.py

```diff
@@ -97,7 +101,7 @@ class Accessor:
     .. versionadded:: 2.0.0
     """
 
-    def __init__(self, name, accessor):
+    def __init__(self, name: str, accessor: Accessor) -> None:
```

Comment:
This one looks tricky to annotate. I need to give it a deeper look, but I do not have the time today. I'll go back to it either tomorrow or on Wednesday.